bazel/grpc_deps.bzl

```diff
@@ -247,13 +247,16 @@ def grpc_deps():
         )
 
     if "com_github_google_benchmark" not in native.existing_rules():
+        # Release 2021-09-07
+        BM_COMMIT=0baacde3618ca617da95375e0af13ce1baadea47
```

Comment:
let's not introduce the variables for BM_COMMIT and BM_SHA256 since it's inconsistent with the layout of other dependencies in this file.

Btw, we do have a sanity test that looks at the grpc_deps.bzl file and checks whether the submodule versions match the bazel dependency version. I think in this particular case, it might still work, but for the sake of consistency, let's keep the old format (no variables) anyway

Reply:
reverted this change.